zerver/views/realm.py

```diff
@@ -351,9 +353,39 @@ def update_realm(
 
 @require_realm_owner
 @has_request_variables
-def deactivate_realm(request: HttpRequest, user: UserProfile) -> HttpResponse:
+def deactivate_realm(
+    request: HttpRequest,
+    user: UserProfile,
+    realm_deletion_in_minutes: Optional[int] = REQ(
+        json_validator=check_none_or(check_int), default=None
```

Comment:
This API change is missing an API documentation note; we don't have docs for this endpoint so we only need something in the API changelog. I'll fix.

Also renamed to `deletion_delay_minutes` ... though we may want to change the units to days.